public/pages/cooldown/list.html

```diff
@@ -1,76 +1,44 @@
 <div id="systemCooldownApp">
-  <div class="container-fluid">
```

Comment:
Please fix your issue with updating spacing, we can have some refactor commit which is updating spacing, but please let it out of feat commits as I almost cannot track what is changed

Reply:
You can check what was changed in second commit.
But, i have no idea how to fix that spacing. Because i copy original raw file, paster it, and it's still with spacing (git not see any changes after copy paste)